rasa/cli/data.py

```diff
@@ -11,12 +11,16 @@
 from rasa.cli.arguments import data as arguments
 from rasa.cli.arguments import default_arguments
 import rasa.cli.utils
+import rasa.nlu.convert
+from rasa.nlu.data_augmentation import augmenter
```

Comment:
should be a local import as this import tensorflow top-level (`test_cli_start_is_fast` is failing due to that) which in turn causes our CLI to be slow

Reply:
the `augmenter` import + `rasa.nlu.test`

Reply:
changed the `augmenter` import and removed the `rasa.nlu.test` import - didn't look like it was used by anything in `rasa/cli/data.py`